code/game/objects/machinery/deployable.dm

```diff
@@ -59,7 +59,7 @@
 	if(!item)
 		return
 	if(CHECK_BITFIELD(item.flags_item, DEPLOYED_NO_PICKUP))
-		to_chat(user, span_notice("The [src] is anchored in place and cannot be disassembled."))
+		balloon_alert(user, "Cannot disassemble")
```

Comment:
you should probably say *why* its not undeplayoable instead

Reply:
It's undeployable because it has the flag.

Reply:
"[src] is undeployable because it cannot be undeployed"

Reply:
ok but it said anchored before was that lying

Reply:
Yes.

Reply:
Wether you anchor the tat or not, you can still NEVER undeploy it. This flag is not added/removed anywhere other than on the define itself.